examples/layer-browser/src/examples/additional-layers.js

```diff
@@ -15,12 +15,31 @@ import * as h3 from 'h3-js';
 
 import {registerLoaders} from '@loaders.gl/core';
 import {PLYLoader} from '@loaders.gl/ply';
-import {GLTFScenegraphLoader} from '@luma.gl/addons';
+import {GLTFScenegraphLoader, GLTFEnvironment} from '@luma.gl/addons';
+import GL from '@luma.gl/constants';
 
 import * as dataSamples from '../data-samples';
 
 registerLoaders([GLTFScenegraphLoader, PLYLoader]);
 
+const GLTF_BASE_URL =
```

Comment:
This is a lot of scaffolding, maybe split out a `mesh-layers.js` file? It seems reasonable that we have one examples file per module.

Reply:
OK.

Reply:
Done